packages/flutter/lib/src/material/material.dart

```diff
@@ -350,7 +350,7 @@ class _MaterialState extends State<Material> with TickerProviderStateMixin {
     );
     Widget contents = widget.child;
     if (contents != null) {
-      contents = AnimatedDefaultTextStyle(
+      contents = AnimatedDefaultTextStyle.merge(
```

Comment:
This was added because Material was previously overwriting any manually set DefaultTextStyle applied above it instead of merging with it.

Reply:
cc @johnsonmh